dbbackup/db/base.py

```diff
@@ -16,7 +16,8 @@
     'django.db.backends.postgresql': 'dbbackup.db.postgresql.PgDumpConnector',
     'django.db.backends.postgresql_psycopg2': 'dbbackup.db.postgresql.PgDumpConnector',
     'django.db.backends.oracle': None,
-    'django_mongodb_engine': 'dbbackup.db.mongo.MongoDumpConnector',
+    'django_mongodb_engine': 'dbbackup.db.mongodb.MongoDumpConnector',
+    'djongo': 'dbbackup.db.mongodb.MongoDumpConnector',
```

Comment:
djongo ?

Reply:
it is correct. djongo is a better supported mongo db engine for django

Reply:
I know strange name ;)